src/org/usfirst/frc/team5590/robot/commands/CloseGrabber.java

```diff
@@ -0,0 +1,42 @@
+package org.usfirst.frc.team5590.robot.commands;
+
+import org.usfirst.frc.team5590.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
```

Comment:
Please provide a short description of what it does. Look at JoystickDrive for an example.